Doc/library/crypt.rst

```diff
@@ -93,7 +93,7 @@ The :mod:`crypt` module defines the following functions:
    may be available on all platforms), or a full encrypted password
    including salt, as returned by this function.  If *salt* is not
    provided, the strongest method will be used (as returned by
```

Comment:
If `methods` is not a function, do we need the wording `as returned by` ? We usually use return for results of functions and methods. Am assuming due diligence, that `methods` is actually an attribute, not a method, therefore someone else can confirm on this.